core/templates/dev/head/css/oppia.css

```diff
@@ -952,6 +952,10 @@ textarea {
   color: rgba(255,255,255,1);
 }
 
+.btn:focus, .btn:active:focus {
+    outline: 0px;
```

Comment:
I think it's a default browser thing. It's was due to a css in third-party. I'll change it to just 0.